cmd/client/main.go

```diff
@@ -1,10 +1,16 @@
 package main
 
 import (
-	"github.com/ASeegull/SmartFridge/examplePackage"
 	"fmt"
+	"log"
+	"net/http"
+
+	s "github.com/ASeegull/SmartFridge/staticServer"
 )
 
 func main() {
-	fmt.Println(examplePackage.Func("client"))
+	r := s.NewRouter()
+	port := ":" + s.GetPort()
+	fmt.Printf("Client server is listening on port %s\n", port)
```

Comment:
Please use logging packages to write logs. I prefer to use [this](https://github.com/sirupsen/logrus) or [this](https://github.com/uber-go/zap) one. First is quite simpler and would be enough.